app/assets/stylesheets/components/_carousel.scss

```diff
@@ -2,6 +2,21 @@
   position: relative;
 }
 
+.carousel-item-single-page {
+  margin-bottom: 1.5rem;
+  text-align: right;
+
+  a {
+    font-size: $h5-font-size;
+    text-decoration: none;
+
+    &:focus,
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
```

Comment:
<img width="740" alt="screen shot 2017-08-04 at 4 13 59 pm" src="https://user-images.githubusercontent.com/27780860/28985328-01dac3e6-7930-11e7-9f4f-94cf6a92da96.png">

☝️  Styles the "View All in Single Page" link.